services/inc/src/main/java/org/cryptimeleon/incentive/services/promotion/BasketNotPayedException.java

```diff
@@ -0,0 +1,12 @@
+package org.cryptimeleon.incentive.services.promotion;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(HttpStatus.BAD_REQUEST)
+public class BasketNotPayedException extends RuntimeException {
```

Comment:
Paid?

Reply:
Thanks for pointing that out. Fixed the typo everywhere